data_steward/analytics/cdr_ops/vocabulary_upload_qc.py

```diff
@@ -132,7 +155,24 @@
                    vocabulary_dataset_new=vocabulary_dataset_new)
 execute(client, query, max_rows=True)
 
-# Shows the number of individual concepts added to each vocab type.The 'diff' should always increase.
+# # Confirm presence of vocabularies 
+# > - This table should contain five rows.<br>
+# > - These vocabulary_ids should exist in the new vocabulary <br>
+# > - Investigate if this is not the case. <br>
```

Comment:
Do you know why this specific check exists and why it only focuses on these 5 vocabularies?  If so, that would be good information to add to this description.

Reply:
I remember the TypeConcept and Condition Status vocabs causing trouble.  They were removed and added back inadvertently or something similar.  I'm not sure we have to have this particular query anymore since we  do have the above query.  Perhaps you could do a `FULL OUTER JOIN` on the previous query to identify all vocab changes from old to new vocabularies and highlight anything that either didn't match or where a `null` value existed.  If nothing is changed, we don't actually have to see the full 80-ish results list.  Thoughts?

Reply:
I removed this query, and made a small change to the one above it. I also thought it unnecessary to see the entire vocabulary list.